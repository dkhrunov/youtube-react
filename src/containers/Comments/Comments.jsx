import React from 'react';
import CommentsHeader from './CommentsHeader/CommentsHeader';
import AddComment from './AddComment/AddComment';
import Comment from './Comment/Comment';
import './Comments.scss';

const Comments = (props) => {
	return (
		<div>
			<CommentsHeader amountComments={props.amountComments} />
			<AddComment />
			<Comment />
			<Comment />
			<Comment />
		</div>
	);
};

export default Comments;
