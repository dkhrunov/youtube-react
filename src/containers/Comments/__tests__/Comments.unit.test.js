import React from 'react';
import { shallow } from 'enzyme';
import Comments from '../Comments';

describe('Renders Comments component', () => {
	test('renders without props', () => {
		const wrapper = shallow(<Comments />);
		expect(wrapper).toMatchSnapshot();
	});

	test('renders with amountComments', () => {
		const wrapper = shallow(<Comments amountComments={666} />);
		expect(wrapper).toMatchSnapshot();
	});
});
