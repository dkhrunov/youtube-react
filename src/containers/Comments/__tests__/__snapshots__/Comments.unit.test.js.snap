// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Comments component renders with amountComments 1`] = `
<div>
  <CommentsHeader
    amountComments={666}
  />
  <AddComment />
  <Comment />
  <Comment />
  <Comment />
</div>
`;

exports[`Renders Comments component renders without props 1`] = `
<div>
  <CommentsHeader />
  <AddComment />
  <Comment />
  <Comment />
  <Comment />
</div>
`;
